core/capabilities/syncer.go

```diff
@@ -98,36 +104,42 @@ func (s *registrySyncer) Start(ctx context.Context) error {
 	}
 	myId := s.peerWrapper.GetPeer().ID().String()
 	config := remotetypes.RemoteTriggerConfig{
-		RegistrationRefreshMs: 20000,
+		RegistrationRefreshMs:   20000,
+		MinResponsesToAggregate: 2,
```

Comment:
```suggestion
		MinResponsesToAggregate: triggerCapabilityDonInfo.F*2+1,
```
Shouldn't this be 2F_Remote+1? I'm going off of the [remote trigger DD spec](https://docs.google.com/document/d/1Iy5tU8Lq28x5PiMOCE18yeaRyip8gWzij2DfFL2bhcY/edit#heading=h.5jz6kr4eo0ne)

Reply:
For Mercury reports it's enough to wait for F+1 instead of 2F+1 because the aggregator is smart and will pick latest reports from each message individually. I'll change to using config vars anyway, thanks.